### PR TITLE
Enable CI tests.

### PR DESCRIPTION
**Pull request by cryptcoin-junkey**

### CODE REVIEW

File: .github/workflows/python-app.yml [hunk 1]
```diff
@@ -37,4 +39,7 @@ jobs:
         flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
     - name: Build
       run: |
-        python setup.py install
+        python setup.py develop
+        python setup.py install_apsw
+    - name: Test
+      run: pytest
```
Comment (wakiyamap) [inline .github/workflows/python-app.yml]: run: pytest
->
run: python -m pytest
is better.

**Comment by wakiyamap** [comment]
I want to help with the testing. 
Which branch should I send a pull request?

**Comment by cryptcoin-junkey** [comment]
Looks many tests failed. They seem to be caused by differences between Monacoin and Bitcoin. Not degrades.

**Comment by cryptcoin-junkey** [comment]
> Which branch should I send a pull request?

~~Please request to this branch.~~

I'm going to merge this to `monaparty-develop` branch.
Could you send your patches based on `monaparty-develop`? @wakiyamap

**Pull request merged**